### DISCUSSION TITLE
Efficient Multiple Bots?

### DISCUSSION THREAD

**Discussion by anarchy1337x** [PrismarineJS/mineflayer#1950] [Q&A]
I've been working with the mineflayer api for about 2 days or so, and i really like it. It's exactly what i need to perform the stuff i do.

One issue is multiple accounts, i've looked at "multiple.js" and the pathfinding "multiple.js" and none of them seem to work.

The threads seem to spin up, but just exit once done.
I've already configured the server ip etc, but none seem to work

Has anyone got any referenced on how to do this?

What i'm trying to do is:

- Spin up multiple bots
- Control all the bots at once / or control one bot at a time.

**Comment by rom1504** [comment]
Could you share the exact code you're running ?

**Comment by Hazuki-san** [comment]
I think using child-process might be a good idea for it, try look at my code https://github.com/Hazuki-san/HoapBot
